### ISSUE TITLE
Pass attributes to processor.load in getData 

### ISSUE THREAD

**Issue by eiswind** [saneef/eleventy-plugin-asciidoc#16]
When including source code examples I rely on a custom path attribute like:

```
[source,java]
----
include::{examplesdir}/spring/restclienttest/src/main/java/de/eiswind/restclienttest/SomeWildApiClient.java[tag=snip]
----
<1> some callout text
```
this works fine, but i get warnings in getData about callouts not being found, and my guess would be that the examplesdir attribute is not defined when processer.load gets called. 

I could try to create a PR if you agree and pass available ProcessorOptions to load.

**Comment by saneef** [comment]
Thanks for reporting. I have published v3.1.2 with fix for this issue.

**Comment by eiswind** [comment]
Thx! Just updated, no more warnings! :)
